resources/Microsoft.VSCode.Dsc/Microsoft.VSCode.Dsc.psm1

```diff
@@ -106,115 +114,124 @@ class VSCodeExtension
     [DscProperty()]
     [bool] $Exist = $true
 
+    [System.Management.Automation.HiddenAttribute()]
+    [DscProperty()]
+    [System.Boolean]
+    $UseInsiders = $false
+
+    [System.Management.Automation.HiddenAttribute()]
+    [DscProperty()]
+    [System.String]
+    $CustomPath
+
     static [hashtable] $InstalledExtensions
 
-    static VSCodeExtension()
-    {
-        [VSCodeExtension]::GetInstalledExtensions()
+    VSCodeExtension() {
     }
```

Comment:
I'm okay with moving GetInstalledExtensions() to the get method. Since we are no longer using a static constructor, this can be removed.

Reply:
@ryfu-msft  I didn't totally get what you mean what is required to be removed.